awx/main/models/ha.py

```diff
@@ -354,7 +354,7 @@ def local_health_check(self):
         except redis.ConnectionError:
             errors = _('Failed to connect to Redis')
 
-        self.save_health_data(awx_application_version, get_cpu_count(), get_mem_in_bytes(), update_last_seen=True, errors=errors)
+        self.save_health_data(self, awx_application_version, get_cpu_count(), get_mem_in_bytes(), update_last_seen=True, errors=errors)
```

Comment:
`self` is automatically passed by python as the first argument in bound methods like these.